joern-cli/frontends/rubysrc2cpg/src/test/scala/io/joern/rubysrc2cpg/passes/ast/SimpleAstCreationPassTest.scala

```diff
@@ -903,5 +903,39 @@ class SimpleAstCreationPassTest extends RubyCode2CpgFixture {
       assocCallNode.argument.argumentIndex(1).head.code shouldBe "if"
       assocCallNode.argument.argumentIndex(2).head.code shouldBe "x.nil?"
     }
+
+    "have correct structure for proc definiton with procParameters and empty block" in {
+      val cpg =
+        code("-> (x,y) {}")
+      cpg.parameter.size shouldBe 3
```

Comment:
Could you explain why 3? The sample seems to suggest 2 parameters. As a suggestion, maybe also include the names of parameters in the assertion?

Reply:
This is because when AST For parameters is called, a implicit `this` parameter is added.